test/winter_onboarding_2021/fleet_management/db/user_test.clj

```diff
@@ -0,0 +1,35 @@
+(ns winter-onboarding-2021.fleet-management.db.user-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.db.user :as user-db]
+            [winter-onboarding-2021.fleet-management.fixtures :as fixtures]))
+
+(use-fixtures :once fixtures/config fixtures/db-connection)
+(use-fixtures :each fixtures/clear-db)
+
+(deftest create-user
+  (testing "Should create a user in the users table"
+    (let [user {:name "Harry Potter"
+                :role "admin"
+                :email "harry@hogwarts.edu"
+                :password "hermione@123"}
+          _ (user-db/create user)]
```

Comment:
Why not just directly pass the map to `user-db/create`?

Reply:
Please add more test cases here? What if email is nil, or name is nil? Since this is the DB namespace it should test all the edge cases thoroughly.